R/table.R

```diff
@@ -42,3 +43,108 @@
     }
   )
 }
+
```

Comment:
Add to vignette
- modify example by adding Date
- mention handling date in a new section (reference POSIXlt docs)